simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/FOSS5xPerfMedium.scala

```diff
@@ -0,0 +1,500 @@
+package com.puppetlabs.gatling.node_simulations
+import com.puppetlabs.gatling.runner.SimulationWithScenario
+import org.joda.time.LocalDateTime
+import org.joda.time.format.ISODateTimeFormat
+import java.util.UUID
+
+import scala.concurrent.duration._
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+// import io.gatling.jdbc.Predef._
+
+class FOSS5xPerfMedium extends SimulationWithScenario {
+
+// 	val httpProtocol = http
+// 		.baseURL("https://${node}:8140")
+
+	val reportBody = ELFileBody("FOSS5xPerfMedium_0127_request.txt")
+
+	val headers_0 = Map("X-Puppet-Version" -> "5.0.0")
+	val headers_1 = Map(
+		"X-Puppet-Version" -> "5.0.0",
+		"Accept" -> "application/json",
+		"Content-Type" -> "application/json",
+		"Connection" -> "close")
```

Comment:
I think it would be good to include `Accept-Encoding` header for this, as a real agent would, so that we can have the server exercise gzip encoding for the response bodies it produces.  Here's what my agent sends for this:

```
Accept-Encoding: gzip;q=1.0,deflate;q=0.6,identity;q=0.3
```